JarrydWorland/2021_Capstone_GP2_Team
# Refactor lamp

## Changes include
- Added lamp enemy sprites (382546f)
- Added lamp enemy animations (ea4a3e7)
- Added light-bulb projectile (2d22515)
- Fixed health not being initialized properly (e5e08a0)
- Added lamp enemy (7633514)
    - Added base enemy
    - Added EnemyDamageBehaviour

## Issues
Currently the lamp shoots light-bulbs from a child-object point called "ProjectileSpawn" so that the point position can be changed during animations allowing for the light-bulbs to always spawn from the lamps "mouth". However unity is blending the points between directions and I cant see a way to disable it. This causes the spawn locations to be incorrectly offset, and should be looked into later.

## How to test
Place a level and a player, and attach a CameraPanBehaviour to the default camera. And place some lamp enemies. Ensure the enemies fire at you, and that you can defeat them.
